src/index.ts

```diff
@@ -19,10 +19,29 @@ const displayStackSimple = winston.format((info: TransformableInfo) => {
   return info
 })
 
+const addIndent = (str: string, indentLevel: number): string => {
+  return str.split("\n").map(item => `${'  '.repeat(indentLevel)}${item}`).join("\n")
+}
+
+const mergeStacks = (err: any) => {
+  if (!(err.message instanceof Error)) {
+    return ''
+  }
+  let error = err.message
+  let stack = `${error.stack}`
+  let indent = 1
+  while (error && error.error && error.error instanceof Error && error.error.stack) {
+    stack = `${stack}\n`
+    error = error.error
+    stack = `${stack}${addIndent(error.stack, indent++)}\n`
+  }
+  return stack
+}
+
 const displayStackJson = winston.format((info: any) => {
```

Comment:
The `unknown` type should be preferred over `any` if we don't know what we get, as the type checks get enforced.

Reply:
Yyeah I know but this cause issues in other implementations so I will leave it for now.